src/main/java/frc/robot/commands/testing/TestSubsystem.java

```diff
@@ -0,0 +1,147 @@
+package frc.robot.commands.testing;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.Subsystem;
+import frc.robot.devices.ColorSensor;
+import frc.robot.devices.Lemonlight;
+import frc.robot.devices.LidarV4;
+import frc.robot.oi.drivers.ShuffleboardDriver;
+import frc.robot.utilities.Testable;
+import frc.robot.utilities.lists.Colors;
+import java.util.HashMap;
+
+/**
+ * Tests a testable subsystem.
+ */
+public class TestSubsystem extends CommandBase {
+
+    private CANSparkMax[] motors;
+    private ColorSensor[] colorSensors;
+    private Lemonlight[] limelights;
+    private LidarV4[] lidars;
```

Comment:
Make is a list of Lidars, not LidarV4. Again LidarV4 is the implementation for the specific lidar.